package.json

```diff
@@ -182,6 +185,9 @@
       "ts-jest": {
         "tsconfig": "<rootDir>/src/frontend/tsconfig.json"
       }
+    },
+    "moduleNameMapper": {
+      "nav-(.*)-style": "<rootDir>/src/mockFile.js"
```

Comment:
Hva gjør denne? :)

Reply:
Når man prøver å legge inn tester på komponenter (som drar inn react og styles) så prøver jest å kjøre filene som blir importer, men jest vet ikke hvordan den skal parse less filer f.eks. Derfor er denne fiffige kodebiten (som @KenAJoh bidro med) lagt inn for at hver gang (spesielt nav-frontend less her) jest prøver å kjøre less filer blir den bare "routet" til denne mockfilen istedet :)